crane/template.go

```diff
@@ -0,0 +1,105 @@
+package crane
+
+import (
+	"fmt"
+	"github.com/michaelsauter/crane/print"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+	"text/template"
+)
+
+type TemplateData struct {
+	Graph              DependencyGraph
+	TargetedContainers Containers
+}
+type TemplateDataForContainer struct {
+	Graph              DependencyGraph
```

Comment:
Also, I guess the original `Graph`/`DependencyGraph` naming is a bit off, since it's a structure representing a graph (which happens to be a tree) but expressed "flat".